--- ./src/module.c.org	Mon Jun 10 17:07:37 2002
+++ ./src/module.c	Mon Jun 10 17:01:57 2002
@@ -29,8 +29,13 @@
 #include "sys.h"
 #include "arch.h"
 
+#ifndef MODULE_LICENSE
+#define MODULE_LICENSE(license)
+#endif
+
 MODULE_AUTHOR("Copyright (c) 1999-2001 by Wojciech Purczynski <cliph@linux.pl>");
 MODULE_DESCRIPTION("capsel - Linux kernel 2.2.x/2.4.x security module"); 
+MODULE_LICENSE("GPL");
 char kernel_version[]=UTS_RELEASE;
 
 EXPORT_NO_SYMBOLS;
